graphql-dgs-spring-graphql-starter/build.gradle.kts

```diff
@@ -16,9 +16,5 @@
 
 
 dependencies {
```

Comment:
Consider adding a comment that this will be removed in the future and users should use the dgs-starter instead.

Reply:
done